### PR TITLE
Create `profile/README.md`

### LINKED ISSUE #9: MAINT - Add /profile/README.md
### Context

Now GH supports the creation of a `/profile/README.md` file which serves as the main profile file for the org. 

Once we have our design/identity defined it would be worth adding such file		

### Value and/or benefit

As a newcomer I want to have a top-level file with the following info"

- main repos to keep tabs on
- governance items
- how to contribute/interact with the project

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pavithraes**
This PR adds a description on the [`nebari-dev` GitHub organization landing page](https://github.com/nebari-dev).

Currently, this PR adds some helpful links. I did not link to repositories because we can pin some repositories. The pinned repos will appear right after this README, and include a one-line description.

We can pin the following repositories, in order:
- nebari-dev/nebari
- nebari-dev/nebari-docs (only this for now)
- nebari-dev/nebari-demo
- nebari-dev/nebari-design
- nebari-dev/governance
- nebari-dev/nebari-docker-images

## Reference Issues or PRs
Closes #9

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### Any other comments?

The links in this PR need to be updated after some PRs like the roadmap and Nebari repository rename are merged.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Welcome to Nebari! 👋
+
+<a href="https://nebari.dev">
+    <img src="nebari-banner.svg" alt="A banner showing the Nebari logo witt the tagline 'A data science platform: Built for scale, designed for collaboration.' This is followed by a get started button pointing to Nebari 101"/>
```
Comment (pavithraes) [inline profile/README.md]: For reference: The banner is created using [Google Slides here](https://docs.google.com/presentation/d/1_X-EcpFNzrMSyO74oBBi0do_4bsTQwS5xciLOTLbikY/edit?usp=sharing). Please feel free to comment on the slide directly if needed!

Comment (pavithraes) [reply profile/README.md]: I think we can make the banner shorter in height.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+💬 Have a question about using Nebari? Ask on [our community forum](https://github.com/orgs/nebari-dev/discussions).
+
+💎 Wish to contribute to Nebari? Check out [our contributing guidelines](https://www.nebari.dev/community/#how-to-contribute).
+
+✨ Curious what we're working on? Check out [our roadmap](https://github.com/nebari-dev/governance).
+
+---
+
+<!-- We can pin the following repositories, in order:
```
Comment (trallard) [inline profile/README.md]: Maybe for now `nebari` `nebari-docs` only?

**Review by trallard — COMMENTED** [review]
Thanks for working on this!

**Comment by trallard** [comment]
@pavithraes  is this ready for merge?

File: profile/README.md (hunk 2 above)
Comment (pavithraes) [reply profile/README.md]: Sounds good!

**Comment by pavithraes** [comment]
@trallard Made some updates, it's ready now!

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Welcome to Nebari! 👋
+
+<a href="https://nebari.dev">
+    <img src="nebari-banner.svg" alt="A banner showing the Nebari logo witt the tagline 'A data science platform: Built for scale, designed for collaboration.' This is followed by a get started button pointing to nebari.dev."/>
```
Comment (trallard) [inline profile/README.md]:
```suggestion
    <img src="nebari-banner.svg" alt="A banner showing the Nebari logo with the tagline 'A data science platform: Built for scale, designed for collaboration.' This is followed by a get started button pointing to nebari.dev."/>
```

**Review by trallard — APPROVED** [review]
minor change needed - otherwise we can merge

**Pull request merged**